docs/index.rst

```diff
@@ -39,16 +45,7 @@ Links
 
 License: BSD-3
 
-.. _astroplan_news:
 
-News
-====
-
-* August 28, 2015: Astroplan **0.1** release
```

Comment:
If you remove this, can you make it clear somehow differently to the user what the latest Astroplan release is?